src/coreclr/tools/aot/ILCompiler.RyuJit/Compiler/RyuJitCompilation.cs

```diff
@@ -43,7 +44,8 @@ public sealed class RyuJitCompilation : Compilation
             ProfileDataManager profileDataManager,
             MethodImportationErrorProvider errorProvider,
             RyuJitCompilationOptions options,
-            int parallelism)
+            int parallelism,
+            bool ignoreUnresolved)
```

Comment:
Instead of introducing a new bool, could you add this as a flag to `RyuJitCompilationOptions`?